app/assets/stylesheets/_honeycrisp-compact.scss

```diff
@@ -94,9 +105,27 @@
       right: $s25;
     }
   }
+
   .slab--padded {
-    padding-top: 1.5rem;
-    padding-bottom: 1.5rem;
+    padding-top: 2rem;
+    padding-bottom: 2rem;
+  }
+
+  .slab--half-padded {
+    padding-top: 1rem;
+    padding-bottom: 1rem;
+  }
+
+  .single-input-form {
+    textarea {
+      margin-bottom: 0;
+    }
+    .form-group {
+      margin-bottom: 2.5rem;
+    }
+    .button {
+      margin-bottom: 0 !important
+    }
   }
```

Comment:
This looks like something that might make sense to break out into it's own component. What do you think?